### ISSUE TITLE
Schema: Add "No Schema" to list of JSON schemas

### ISSUE THREAD

**Issue by thomassandslyst** [redhat-developer/vscode-yaml#991]
### Is your enhancement related to a problem? Please describe.
Other than editing config files there is no way to disable JSON schema checking against yaml files adhoc if your filename happens to catch one of the schema filters.

### Describe the solution you would like
Adding a permanent "No Schema" or "Disabled" or some other wording option to the JSON Schema selection menu to set the file to not check against schemas

### Describe alternatives you have considered
The current alternative is to either completely disable this feature or edit your settings file and manually exclude every file that hits one of the JSON filename schemas.

**Comment by caubechankiu** [comment]
I also had the same problem

**Comment by designermonkey** [comment]
Yes. This is infuriating at times being unable to deselect a schema. To add to your ticket:

Some documents can't be attributed to a schema, and in this case, the extension simply states `No JSON Schema` in the bar at the bottom.

This option should be provided at the top of the selection list to disable the highlighting.

**Comment by vadimvera** [comment]
In the event someone mistakenly selects a JSON schema, is there a way to revert the association back to `No JSON Schema` for the time being until this option is available?

I've search in many places and can't find anything.

**Comment by ntjess** [comment]
In the meantime, I find if I select the `ntangle` option, it complains about far fewer entries.

**Comment by Kyle-sandeman-mrdfood** [comment]
Almost 2 years later and this is still an issue.
I was editing a file that somehow matched the Ansible format and could not get rid of errors without disabling the remote schema feature completely

+1 for "No JSON schema" being selectable
